Apps/Common/test/unit/Delegates/AesCryptoDelegateTests.cs

```diff
@@ -13,6 +13,8 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 // -------------------------------------------------------------------------
+
+// ReSharper disable CollectionNeverUpdated.Local
```

Comment:
We don't need to suppress this; we can resolve it by refactoring.

Reply:
This was in several places and it is specifically testing an empty collection. So it is legitimate and would not ever be updated. Not sure if that can be refactored.

Reply:
Fixed by inlining the collection.